### ISSUE TITLE
Content suggestion: Translatable "Found a problem with this page?" card

### ISSUE THREAD

**Issue by Jalkhov** [mdn/translated-content#6943]
## What is the new suggestion?
Speaking specifically about this part:
![Captura de pantalla 2022-07-16 a las 08 17 17(1)](https://user-images.githubusercontent.com/36505934/179432655-ef578daa-d871-4792-9c12-0527d1068dd2.png)

In the **i10n-es** team we wondered if this element could be translatable. Personally I did a little research and came to the conclusion that it could be.

## Why is it important or useful?
**Is it SUPER-NECESSARY?:** No
**Would it bring anything extremely useful to MDN?:** Maybe...No.

But...Why not do it?

Beyond being a useful and revolutionary change, I am personally more inclined to aesthetics. That the whole page is in Spanish (for example) and then that element is in English... I do not generate belly ache but... Why can not be in language and keep the same line of language?

## Other supporting information

Prosigo a plasmar mi pequeña "investigación" no con intención de hacer creer que descubrí algo nuevo, sino para que aquellas personas que quizás tengan conocimiento en el lengaje en custión tenga un punto de partida.

Lo primero es:

Where does this element come from? Well, from [HERE](https://github.com/mdn/yari/blob/main/client/src/document/organisms/metadata/index.tsx). More specifically from the `Metadata` function.

Already at first glance we clearly cannot use the conventional method of using markdown files (or can we?) to translate this element.

Before we go on talking about such an element let's look at [this file](https://github.com/mdn/yari/blob/main/client/src/document/molecules/localized-content-note/index.tsx). And that's where the interesting part is, apparently based on the `locale` variable the decision is made as to what text to display. So, back to our main file, I haven't really touched TypeScript in my life, but logic tells me that it is possible to do something similar, so I hope that some Yari connoisseur can guide us and tell us why YES and why not to implement the translation of this element (among others).

PD: I made the decision to label all languages as it is a general issue, any corrections or suggestions are welcome n.n

**Comment by yin1999** [comment]
I thinks we should also consider this mdn/yari#6515

**Comment by SphinxKnight** [comment]
For cross-reference: https://github.com/mdn/yari/pull/6282 / https://github.com/mdn/yari/issues/1197

**Comment by SphinxKnight** [comment]
And as a personal point of view: yes this should be localizable. As of now, it creates a discrepancy when reading a localized page :(

**Comment by queengooborg** [comment]
I agree that this should be localizable as well!  I've opened mdn/yari#6282 to implement localization support into the Yari engine, starting with the BCD tables.  There has been an ongoing discussion on whether to allow the MDN interface to be localizable (there's mixed responses), and I'm waiting to hear back on the decision.  If the decision is "yes" and the PR I've opened is merged, I'll be sure to localize more elements, including this card!

**Comment by SphinxKnight** [comment]
Removing the label for `fr` as there is not much we can do here :/
